src/pages/api/data/dto/GetArticleResponseDto.ts

```diff
@@ -0,0 +1,18 @@
+import { Nullable } from '@common/utils';
+import { Article, Series, User } from '../model';
+
+export interface GetArticleResponseDto {
+    id: number,
+    authorId: number,
+    seriesId: Nullable<number>,
+    author: User,
+    series: Nullable<Series>
+    title: string,
+    content: string,
+    thumbnail: string,
+    viewCount: number,
+    isPublic: boolean;
+    createdAt: Date,
+    updatedAt: Date,
+    deletedAt: Nullable<Date>,
+};
```

Comment:
```suggestion
    id: number;
    authorId: number;
    seriesId: Nullable<number>;
    author: User;
    series: Nullable<Series>;
    title: string;
    content: string;
    thumbnail: string;
    viewCount: number;
    isPublic: boolean;
    createdAt: Date;
    updatedAt: Date;
    deletedAt: Nullable<Date>;
};
```
콤마를 세미콜론으로 바꿔주셔야할 것 같아요!